samples/charts/category-chart/line-chart-with-animations/App.razor

```diff
@@ -0,0 +1,46 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+    <div class="igOptions">
+        <span class="igLegend-title">Renewable Electricity Generated</span>
+        <div class="igLegend">
+            <Legend @ref="LegendRef" Orientation="LegendOrientation.Horizontal" />
+        </div>
```

Comment:
add a combobox to change TransitionIn modes
add a slider to change duration of TransitionIn animation (50ms - 500ms)
add a button to replay animation by forcing the chart to refresh

Reply:
pending